src/timeline/InputBar.cpp

```diff
@@ -341,20 +390,10 @@ InputBar::send()
 
     auto wasEdit = !room->edit().isEmpty();
 
-    if (text().startsWith('/')) {
-        int command_end = text().indexOf(QRegularExpression(QStringLiteral("\\s")));
-        if (command_end == -1)
-            command_end = text().size();
-        auto name = text().mid(1, command_end - 1);
-        auto args = text().mid(command_end + 1);
-        if (name.isEmpty() || name == QLatin1String("/")) {
-            message(args);
-        } else {
-            command(name, args);
-        }
-    } else {
+    if (auto [commandName, args] = getCommandAndArgs(); commandName.isNull())
```

Comment:
Please just check for isEmpty(), I don't see a benefit of using isNull here and it just makes me ask questions :D

Reply:
I kinda assumed that isNull() and isEmpty() were the same. Will fix

Reply:
Nope. And if they were, I would still tell you to use isEmpty, because it is clearer and we use it elsewhere. The difference is actually documented and isEmpty recommended: https://doc.qt.io/qt-6/qstring.html#isNull